tests/testMainFlow.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import pandas as pd
+import sys
+import mlflow
+import json
+import warnings 
+warnings.filterwarnings('ignore')
+sys.path.append('../')
+from src.utils.functions import main_flow
+
+
+test_path = '/mnt/c/Users/Dev/desktop/mlops_project/tests/wine_data/test_wine_data.csv'
+train_path = '/mnt/c/Users/Dev/desktop/mlops_project/tests/wine_data/train_wine_data.csv'
+
+
+class TestMainFlow(unittest.TestCase):
+
+    def setUp(self):
+        print('test started')
+
+    def test_mainflow(self):
+        training_path = train_path
+        val_path = test_path
```

Comment:
This is a code start but given the number of tests you need to run this, all your tests should be have passed into a single class instead of different classes in separate files